source/documentation/deploying_services/elasticsearch.md

```diff
@@ -269,13 +269,13 @@ cf update-service my-es-service -c '{"ip_filter": "1.2.3.4"}'
 
 #### Removing IP addresses from an existing service
 
-To remove all IP addresses except the mandatory PaaS ones, pass an empty value in for the `ip_filter` key:
+To remove all IP addresses except the mandatory GOV.UK PaaS ones, pass an empty value in for the `ip_filter` key:
 
 ```
 cf update-service my-es-service -c '{"ip_filter": ""}'
 ```
 
-To remove one IP address from a list while keeping the mandatory PaaS ones, pass in the list without the IP address you wish to remove for the `ip_filter` key, for example removing `5.6.7.8` from the list of `1.2.3.4, 5.6.7.8, 9.10.11.12`:
+To remove one IP address from a list while keeping the mandatory GOV.UK PaaS ones, pass in the list without the IP address you wish to remove for the `ip_filter` key, for example removing `5.6.7.8` from the list of `1.2.3.4, 5.6.7.8, 9.10.11.12`:
 
 ```
 cf update-service my-es-service -c '{"ip_filter": "1.2.3.4,9.10.11.12"}'
```

Comment:
unrelated, but it makes me so sad that this isn't a list ["1.2.3.4","9.10.11.12"]